### PR TITLE
Support fig.yml

### LINKED ISSUE #864: Support fig.yml and show a deprecation warning

### PR DESCRIPTION
**Pull request by aanand**
Closes #864.

### CODE REVIEW

File: compose/cli/command.py [hunk 1]
```diff
@@ -110,13 +107,34 @@ def get_config_path(self, file_path=None):
...
+        if winner == 'docker-compose.yaml':
             log.warning("Please be aware that .yml is the expected extension "
                         "in most cases, and using .yaml can cause compatibility "
-                        "issues in future")
+                        "issues in future.\n")
 
-            return os.path.join(self.base_dir, 'docker-compose.yaml')
+        if winner.startswith("fig."):
+            log.warning("%s is deprecated and will not be supported in future. "
+                        " Please use docker-compose.yml from now on.\n" % winner)
```
Comment (thaJeztah) [inline compose/cli/command.py]: nit: perhaps "Please rename your config file to `docker-compose.yml`" (or similar)?

**Comment by thaJeztah** [comment]
Thanks for this, much appreciated!

File: compose/cli/command.py (hunk 1 above)
Comment (bfirsh) [inline compose/cli/command.py]: :+1:

Comment (aanand) [inline compose/cli/command.py]: Done.

**Comment by thaJeztah** [comment]
Message looks ok to me.

There's a leading space before the message, I don't think that's intentional?

(On mobile, so can't comment inline, sorry!)

**Comment by aanand** [comment]
Well spotted. Fixed.

**Comment by thaJeztah** [comment]
Thanks! Looks good to me, but I'm not very good at Python, so I may have missed things :)

Final nit/question; you changed this test;

     	command.base_dir = 'tests/fixtures/Simple-figfile'

The new version doesn't have a dash (`-`); are paths containing dashes already covered by other tests?

**Comment by aanand** [comment]
Yep - https://github.com/aanand/fig/blob/support-fig-yml/tests/unit/cli_test.py#L18-L33

**Comment by dnephin** [comment]
LGTM

**Comment by thaJeztah** [comment]
Alright!  (Searching GitHub on mobile is a disaster LOL).

Looks good, thanks!

**Pull request merged**